client/my-sites/checkout/composite-checkout/payment-method-helpers.js

```diff
@@ -89,7 +88,7 @@ export async function createAccount( { signupFlowName } ) {
 				extra: { username_hint: blogName },
 				signup_flow_name: signupFlowName,
 				validate: false,
-				ab_test_variations: getSavedVariations(),
+				ab_test_variations: {},
```

Comment:
Can be removed completely.

Reply:
Removed in 45a754fdee6b5f935552459e1fd1e937587aead8